### DISCUSSION TITLE
Is is suitable for Billing software?

### DISCUSSION THREAD

**Discussion by Akash9630** [KevinVandy/material-react-table#338] [Q&A]
Hi...All, I am planning to build a Jewel Billing System, so basically list of products to be displayed with CRUD opertions, reports to export and Invoice with edit option and printable. please guide me possible outcomes

**Comment by KevinVandy** [comment]
Look at the comparison tables on the docs home page for more info. I can't make the decision for you, but editing is supported in MRT. https://www.material-react-table.com/#let's-compare
